### DISCUSSION TITLE
Floor slab

### DISCUSSION THREAD

**Discussion by Aliturkisk** [BriefFiniteElementNet/BriefFiniteElement.Net#171] [Q&A]
Is there a sample of solving shell element to represent a floor slab?

**Comment by epsi1on** [comment]
There are two elements, `TriangleElement` and `QuadrilaturalElement` both are planar elements which can have membrane and PlateBending behaviors if you set (have a look at [here](https://bfenet.readthedocs.io/en/latest/elements/finiteElements/Triangle/index.html)).
Not sure which behaviour meets your need (concrete floor slab). 
I think it also depends on the slab you are trying to model, two way or single way for examples etc...
All options you have is thickness of `TriangleElement`, Material and `TriangleElement.Behaviour`.

**Reply by Aliturkisk** [reply]
Thank you for reply. These means I make meshing for slab and define it as QuadrilaturalElement?

**Reply by epsi1on** [reply]
I mean you must know which behaviour of triangle or quad element, meets your needs.

**Comment by epsi1on** [comment]
inactivity
